test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -17,6 +17,7 @@ class test_ucp_ep_reconfig : public ucp_test {
 protected:
     using address_t = std::pair<void*, ucp_unpacked_address_t>;
     using address_p = std::unique_ptr<address_t, void (*)(address_t*)>;
+    using limits    = std::numeric_limits<unsigned>;

```

Comment:
As we discussed, maybe using `0` instead can be simpler.

Reply:
seems `0` is used by the test, so it cannot be a special value